stdlib/2and3/binhex.pyi

```diff
@@ -0,0 +1,53 @@
+import sys
+from typing import (
+    Any,
+    Tuple,
+    Union,
+)
+
+
+class Error(Exception): ...
+
+REASONABLY_LARGE = ... # type: int
+LINELEN = ... # type: int
+RUNCHAR = ... # type: bytes
+
+class FInfo:
+    def __init__(self) -> None: ...
+    Type = ... # type: str
+    Creator = ... # type: str
+    Flags = ... # type: int
+
+_FileInfoTuple = Tuple[str, FInfo, int, int]
+
+if sys.version_info < (3, 0):
+    _FileHandleUnion = Union[str, file]
+else:
+    from io import FileIO
+    _FileHandleUnion = Union[str, FileIO]
+
+def getfileinfo(name: str) -> _FileInfoTuple: ...
+
+class openrsrc:
+    def __init__(self, *args: Any) -> None: ...
+    def read(self, *args: Any) -> bytes: ...
+    def write(self, *args: Any) -> None: ...
+    def close(self) -> None: ...
+
+class BinHex:
+    def __init__(self, name_finfo_dlen_rlen: _FileInfoTuple, ofp: _FileHandleUnion) -> None: ...
+    def write(self, data: bytes) -> None: ...
+    def close_data(self) -> None: ...
+    def write_rsrc(self, data: bytes) -> None: ...
+    def close(self) -> None: ...
+
+def binhex(inp: str, out: str) -> None: ...
+
+class HexBin:
+    def __init__(self, ifp: _FileHandleUnion) -> None: ...
+    def read(self, *n: int) -> bytes: ...
```

Comment:
The implementation of this function makes me think that it dates from before Python had default arguments.